### PR TITLE
presidio ignore

### PR DESCRIPTION
**Pull request by arkadiuszbeer**
Issue:
Checking .svg, Rd git files
https://github.com/insightsengineering/teal.widgets/runs/5462808447?check_suite_focus=true#step:3:752

Test:
https://github.com/insightsengineering/teal.widgets/runs/5463258506?check_suite_focus=true

### CODE REVIEW

File: .github/workflows/gitleaks.yaml [hunk 1]
```diff
@@ -69,5 +69,14 @@ jobs:
         with:
           configuration-data: |
             threshold: 0.95
+            ignore: |
+              .git
+              man/figures/*.svg
```
Comment (tomszosz) [inline .github/workflows/gitleaks.yaml:74]: could you change it to **/*.svg ?
could you change it to **/*.png ?

File: .github/workflows/gitleaks.yaml [hunk 2]
```diff
@@ -69,5 +69,14 @@ jobs:
...
+              man/figures/*.png
+              DESCRIPTION
```
Comment (tomszosz) [inline .github/workflows/gitleaks.yaml:76]: I expect, that this file should be checked

Comment (arkadiuszbeer) [reply .github/workflows/gitleaks.yaml]: I have removed `DESCRIPTION` from ignore statement.

**Review by tomszosz — APPROVED** [review]
lgtm

File: .github/workflows/gitleaks.yaml (hunk 1 above)
Comment (arkadiuszbeer) [reply .github/workflows/gitleaks.yaml]: `**/*.png` and `**/*.svg`

**Pull request merged**